lib/mollie/order/line.rb

```diff
@@ -0,0 +1,55 @@
+module Mollie
+  class Order
+    class Line < Base
+      attr_accessor :id, :order_id, :type, :name, :status, :is_cancelable,
+                    :quantity, :quantity_shipped, :amount_shipped,
+                    :quantity_refunded, :amount_refunded, :quantity_canceled,
+                    :amount_canceled, :unit_price, :discount_amount,
+                    :total_amount, :vat_rate, :vat_amount, :sku, :image_url,
+                    :product_url, :created_at
```

Comment:
There are 3 (undocumented) properties: https://github.com/mollie/api-documentation/pull/255/files#diff-ed37081d1c53f913b1b2d7f4ae87e64fR130. There's a PR to update the documentation.

- shippableQuantity
- refundableQuantity
- cancelableQuantity